src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -605,61 +605,103 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColumnInfo info, string
         }
 
         /// <summary>
-        /// An interface implemented by items of <see cref="ColumnFunctions"/> corresponding to the
-        /// <see cref="NormalizeTransform.AffineColumnFunction"/> items.
+        /// Base class for all the NormalizerData classes: <see cref="AffineNormalizerModelParameters{TData}"/>,
+        /// <see cref="BinNormalizerModelParameters{TData}"/>, <see cref="CdfNormalizerModelParameters{TData}"/>.
         /// </summary>
-        internal interface IAffineData<TData>
+        public abstract class NormalizerModelParametersBase
+        {
+        }
+
+        /// <summary>
+        /// The model parameters generated by affine normalization transformations, like
+        /// <see cref="T:Microsoft.ML.StaticPipe.NormalizerStaticExtensions.Normalize{Vector, bool, long, OnFitAffine}"/>,
+        /// <see cref="T:Microsoft.ML.StaticPipe.NormalizerStaticExtensions.NormalizeByMeanVar{Vector, bool, bool, long, OnFitAffine}"/>
+        /// or methods in the <see cref="NormalizerCatalog"/> having the <see cref="NormalizingEstimator.NormalizerMode"/> parameter set to either
+        /// <see cref="NormalizingEstimator.NormalizerMode.MinMax"/> or <see cref="NormalizingEstimator.NormalizerMode.MeanVariance"/>.
+        /// </summary>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Normalize](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Normalizer.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public sealed class AffineNormalizerModelParameters<TData> : NormalizerModelParametersBase
```

Comment:
>TData [](start = 60, length = 5)

It seems like enumerating the types this thing *might* be (since there are only four conceivable types) would go a long way towards making this and the other classes more comprehensible.